.github/workflows/images-ghcr.yaml

```diff
@@ -0,0 +1,172 @@
+name: Build and Publish Retina Container Images [GHCR]
```

Comment:
Don't think we should maintain two `images*.yaml` files; I think we should update `IMAGE_REGISTRY`
in the make file to make ghrc.io the default -
https://github.com/microsoft/retina/blob/91de0eb612910a61568cb4f2a84affdcf6ce70ce/Makefile#L31

Reply:
updated